Framework/Algorithms/inc/MantidAlgorithms/SampleCorrections/RectangularBeamProfile.h

```diff
@@ -30,14 +30,23 @@ class MANTID_ALGORITHMS_DLL RectangularBeamProfile final : public IBeamProfile {
   IBeamProfile::Ray generatePoint(Kernel::PseudoRandomNumberGenerator &rng,
                                   const Geometry::BoundingBox &bounds) const override;
   Geometry::BoundingBox defineActiveRegion(const Geometry::BoundingBox &) const override;
+  /// Returns the min point of the profile
+  inline Kernel::V3D minPoint() const { return Kernel::V3D{m_min[0], m_min[1], m_min[2]}; }
```

Comment:
Now that `m_min` is a `Kernel::V3D`, can we just return `m_min` here?
```suggestion
  inline Kernel::V3D minPoint() const { return m_min; }
```

Reply:
For some reason, m_min is defined as a std::array<double, 3> so I don't think it will work if I just return m_min. I don't see a good reason for not defining m_min as a V3D but I guess I decided to leave this alone given that I was just adding a couple of get methods to this class.